.buildscript/push_javadocs.sh

```diff
@@ -0,0 +1,20 @@
+if [ "$TRAVIS_REPO_SLUG" == "uber/simple-store" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_BRANCH" == "master" ]; then
+
+  echo -e "Publishing javadoc...\n"
+
+  cp -R build/docs/javadoc $HOME/javadoc-latest
+
+  cd $HOME
+  git config --global user.email "travis@travis-ci.org"
```

Comment:
Is this the right user?

Reply:
@briankhsieh any thoughts on using an email here?

Reply:
you can ci@uber.com